.github/workflows/manual-artifacts-generation.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           path: exporters/${{ inputs.integrationName }}/target/packages/*
 
   build_artifacts_windows:
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
this scares me a bit , taking into account that in the pws scripts we look installed apps in certain paths IIRC, but if you feel confident, and is not breaking 👍

Reply:
I have mixed feelings with this.

It broke in the past so Marc had to change it: https://github.com/newrelic/newrelic-prometheus-exporters-packages/commit/f1d03c7555e29e57c3af119b79114ca7b9753c77#diff-18a39f21aef81bfebafb618d7701ab708ca8dfe6432217dbb53959ea31076f79R11

I prefer to have a pipeline updated that breaks and we have to fix it that having an outdated that we know it is reproducible but in the future will break in a bigger way. But this is my point of view. Let me know your thought on this and I might change my mind :)

Reply:
Yes it make sense, perhaps having renovate bumping this versions on PR could make a middle ground, helping us to keep it update it but without many breaks WDYT?
 . I'm not sure if Renovate can bumps runners though